### PR TITLE
for #333 - access _default_manager via TagBase class

### PR DESCRIPTION
**Pull request by groovecoder**
[As suggested via @jezdez in #334](https://github.com/alex/django-taggit/pull/334#discussion-diff-36459222).

### CODE REVIEW

File: taggit/models.py [hunk 1]
```diff
@@ -68,8 +68,9 @@ def save(self, *args, **kwargs):
             except IntegrityError:
                 pass
             # Now try to find existing slugs with similar names
+            import ipdb; ipdb.set_trace()
```
Comment (frewsxcv) [inline taggit/models.py]: :)

Comment (groovecoder) [inline taggit/models.py]: :rage:

**Comment by groovecoder** [comment]
Updated.

**Comment by frewsxcv** [comment]
In case you missed it, the tests failed

**Comment by groovecoder** [comment]
@jezdez - the test failures are what I didn't understand: how to access `_default_manager` of `TagBase` which is an abstract base class.

**Comment by groovecoder** [comment]
Got the `.__class__` method from IRC logs last week. Still 1 test failure I need to debug.

**Comment by groovecoder** [comment]
Updated; everything should be fixed now.

**Comment by groovecoder** [comment]
Any chance to get a merge & release for this anytime soon?

**Comment by frewsxcv** [comment]
Released 0.16.4 with this change

**Pull request merged**